plugins/org.jkiss.dbeaver.ext.postgresql/tests/org/jkiss/dbeaver/ext/postgresql/model/PostgreWithClauseAppenderTest.java

```diff
@@ -0,0 +1,127 @@
+package org.jkiss.dbeaver.ext.postgresql.model;
+
+import org.jkiss.dbeaver.ext.postgresql.model.impls.greenplum.GreenplumTable;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.jkiss.dbeaver.ext.postgresql.model.PostgreWithClauseAppender.generateWithClause;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PostgreWithClauseAppenderTest {
+
+    private PostgreTableRegular table;
+    private PostgreTableBase tableBase;
```

Comment:
You could get rid of mocking table and tableBase in each test using Mock and RunWith(MockitoJunitRunner.class) annotations.

Reply:
Refactored the unit tests to use the MockitoJunitRunner as suggested. This removed the need for a `@Before` annotated setup method.

Some of the unit tests require that `table` be a mock of the `GreenplumTable` class and thus they set directly set this overriding the global mock of PostgreTableRegular.